src/ens/resolvers/index.ts

```diff
@@ -0,0 +1,6 @@
+import * as _ from 'lodash';
+
+import resolvers from './resolvers';
+import scalars from './scalars';
+
+export default [resolvers, scalars].reduce((prev, curr) => _.merge({}, curr, prev), {});
```

Comment:
Take for example the `account` resolver which now needs to be wrapped in the ENS resolution logic.

Instead of returning a resolvers map here, we would return a function like the following:

```
export default (prev) => {
  const wrappedResolvers = {
      Query: { 
        account: (obj, args, context: EthqlContext) => {
          args.address = await args.address(context);
          return prev.Query.account(obj, args, context);
        }
      },
      // same for the other affected resolvers
      // Block: { transactionsInvolving, transactionsRoles },
  };

  return _.mergeDeep(prev, wrappedResolvers);
}
```

Reply:
@raulk looks like i forgot to comment back here after fixing this last week. You can check it now.